src/Cimpress.Auth0.Client/Proxies/RoAuthenticationRequestDto.cs

```diff
@@ -8,14 +8,14 @@ namespace Cimpress.Auth0.Client.Proxies
     /// <remarks>
     /// This has been copied and slightly adapterd from https://github.com/auth0/auth0.net since it doesn't yet support netcore.
     /// </remarks>
-    public class AuthenticationRequestDto
+    public class RoAuthenticationRequestDto
```

Comment:
RO = resource owner

auth0/ro -> log in using username/password authentication
auth0/token -> log in using social providers access token 

I wanted to make sure to differentiate between the various request dtos - if you have a better naming suggestion, I'm open to it, but this was the best that I could find.